Mathlib/RingTheory/UniqueFactorizationDomain/NormalizedFactors.lean

```diff
@@ -220,17 +220,14 @@ theorem zero_notMem_normalizedFactors (x : α) : (0 : α) ∉ normalizedFactors
 @[deprecated (since := "2025-05-23")]
 alias zero_not_mem_normalizedFactors := zero_notMem_normalizedFactors
 
-theorem dvd_of_mem_normalizedFactors {a p : α} (H : p ∈ normalizedFactors a) : p ∣ a := by
-  by_cases hcases : a = 0
-  · rw [hcases]
-    exact dvd_zero p
-  · exact dvd_trans (Multiset.dvd_prod H) (Associated.dvd (prod_normalizedFactors hcases))
+@[deprecated (since := "2025-08-14")]
+alias dvd_of_mem_normalizedFactors := dvd_of_normalized_factor
```

Comment:
Can you swap the deprecation? I believe `dvd_of_mem_normalizedFactors` is more in line with our naming conventions.